Sources/Sharing/SharedReaderKey.swift

```diff
@@ -94,23 +94,21 @@ extension SharedReader {
     wrappedValue: @autoclosure () -> Value,
     _ key: some SharedKey<Value>
   ) {
-    @Dependency(PersistentReferences.self) var persistentReferences
-    self.init(rethrowing: wrappedValue(), key, skipInitialLoad: false)
+    self.init(wrappedValue: wrappedValue(), key)
   }
 
   /// Creates a shared reference to an optional, read-only value using a shared key.
   ///
   /// - Parameter key: A shared key associated with the shared reference. It is responsible for
   ///   loading the shared reference's value from some external source.
-  @_disfavoredOverload
```

Comment:
We should make sure this doesn't introduce ambiguity, but I think it's an accidental disfavor...